docs/github-actions-setup.md

```
@@ -0,0 +1,101 @@
+```
+
+## Troubleshooting
+
+If you encounter the error:
+
+```
+Error: google-github-actions/auth failed with: the GitHub Action workflow must specify exactly one of "workload_identity_provider" or "credentials_json"!
+```
+
+Make sure:
+1. The `GCP_SA_KEY` secret is properly set with the entire JSON content of your service account key
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)